src/wormhole/session/item/rename_gotchi.rs

```diff
@@ -0,0 +1,72 @@
+use super::SessSend;
+use hcor::id::ItemId;
+use hcor::{id, item, Item};
+use std::fmt;
+
+#[derive(Debug)]
+pub enum Error {
+    NoSuch(id::NoSuch),
+}
+use Error::*;
+
+impl From<id::NoSuch> for Error {
+    fn from(ns: id::NoSuch) -> Error {
+        Error::NoSuch(ns)
+    }
+}
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "couldn't spawn items: ")?;
+        match self {
+            NoSuch(e) => write!(f, "no such item: {}", e),
+        }
+    }
+}
+
+pub fn rename(ss: &mut SessSend, item_id: ItemId, new_name: String) -> Result<String, Error> {
```

Comment:
again, nickname would be better here just to match up with GotchiNickname (which I changed so it would be the same as PlantNickname)